freshdesk/scheduled-events/app/js/schedule_detail.js

```diff
@@ -38,36 +38,39 @@ function validateFields() {
 }
 
 function getListOfSchedules(userId, callback) {
-  client.db.get(`schedules_${userId}`)
-  .then(function(data) {
-    callback(data);
-  }, function(err) {
-    if (err.status == 404) {
-      return callback({list:{}});
+  client.db.get(`schedules_${userId}`).then(
+    function (data) {
+      callback(data);
+    },
```

Comment:
You can simplify this code by passing `callback` as the first argument to `.then()` instead of creating a function. For example:

```js
client.db.get(`schedules_${userId}`).then(callback, function (err) {
  // ..handle error
});